CONTRIBUTING.md

```diff
@@ -1,20 +1,191 @@
+
+    ```bash
+    docker run --rm --volume "$(pwd):/terraform-docs" -u $(id -u) quay.io/terraform-docs/terraform-docs:0.17.0 markdown /terraform-docs
+    ```
+If `output.file` is not enabled for this module, generated output can be redirected back to a file:
+
+    ```bash
+    docker run --rm --volume "$(pwd):/terraform-docs" -u $(id -u) quay.io/terraform-docs/terraform-docs:0.17.0 markdown /terraform-docs > doc.md
+    ```
+**Note:** Docker tag `latest` refers to latest stable released version and `edge` refers to HEAD of `master` at any given point in time. And any named version tags are identical to the official GitHub releases without leading `v`.
+
+<sub>[Back to Table of Contents](#table-of-contents)</sub>
+***
+
+## **Fork the repository**
+
+You can fork the hackforla/ops repository by clicking <a href="https://github.com/hackforla/ops/fork"> <button> <img src="https://user-images.githubusercontent.com/17777237/54873012-40fa5b00-4dd6-11e9-98e0-cc436426c720.png" width="8px"> Fork</button></a>
+. A fork is a copy of the repository that will be placed on your GitHub account.
+
+<details>
+  <summary><strong>Click here</strong> to see a visual example of the `fork` button within the website repo</summary>
+  <p><strong>First, you find the `fork` button on the top right hand side of the [Hack for LA website repo](https://github.com/hackforla/website)</strong></h4>
+  <img src="https://user-images.githubusercontent.com/21162229/137044762-c80f18e8-b785-48fb-a286-81c1117c0178.jpg" />
+  <p><strong>Next, a modal may open and request where you want to fork this website repo. Please click on your avatar or your GitHub username</strong></p>
+  <img src="https://user-images.githubusercontent.com/21162229/137045294-3d46b28c-edbb-410c-98f1-13940ecc5c5f.jpg" />
+</details><br>
+
+**Note:** It should create a URL that looks like the following -> `https://github.com/<your_GitHub_user_name>/website`.
+
+**For example** -> `https://github.com/octocat/website`.
+
+**Be Aware:** What you have created is a forked copy in a remote version on GitHub. It is not yet on your local machine yet.
+
+<sub>[Back to Table of Contents](#table-of-contents)</sub>
+***
+
+## **Clone (Create) a copy on your computer**
+
+Create a new folder in your computer that will contain `hackforla` projects.
+
+In your command line interface (Terminal, Git Bash, Powershell), move to where you want your new folder to be placed and create a new folder in your computer that will contain `hackforla` projects. After that, navigate into the folder(directory) you just created.
+
+For example:
+```bash
+mkdir hackforla
+cd hackforla
+```
+
+and run the following commands:
+
+```bash
+git clone https://github.com/<your_GitHub_user_name>/ops.git
+```
+
+For example if your GitHub username was `octocat`:
+```bash
+git clone https://github.com/octocat/ops.git
```

Comment:
```suggestion
git clone https://github.com/octocat/devops-security.git
```

Reply:
The line just below the above code:

```
You should now have a new folder in your `hackforla` folder called `ops`. Verify this by changing into the new directory:

```bash
cd ops
```

```

should be 

```
You should now have a new folder in your `hackforla` folder called `ops`. Verify this by changing into the new directory:

```bash
cd devops-security
```

```